src/transformers/utils/import_utils.py

```diff
@@ -938,6 +968,14 @@ def is_jieba_available():
 Please note that you may need to restart your runtime after installation.
 """
 
+# docstyle-ignore
+MUSIC_IMPORT_ERROR = """
+{0} requires these libraries - pretty_midi, essentia, librosa, scipy. But at least
+one of them was not found in your environment. You can install them with pip:
+`pip install pretty-midi==0.2.9 essentia==2.1b6.dev609 librosa scipy`
```

Comment:
`essentia==2.1b6.dev609` is not available from some environments (if python version is newer than 3.9)
I have verified that `essentia==2.1b6.dev765` works, but I have not verified the following.

1. does it work in newer versions (the latest version is `essentia-2.1b6.dev1034`)
2. whether the beat extraction results in the newer version are the same as in the older version.

I think the answer to both is yes.

Reply:
thanks for the info!